content/showcase/_index.html

```diff
@@ -197,6 +197,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
                         link="https://github.com/sktapsall/CheckerBOT"
                         class="showcase-element has-text-centered"
                     >}}
+                    {{< showcase-item
+                        src="/img/media/showcase/nie.png"
+                        title="UQCS2021-Team-NIE"
```

Comment:
Title should be the name of the project, not the name of the team. See the existing entries at https://uqcs.org/showcase/

Reply:
Changed title to 'Hackathon the Experience'.